src/proto/grpc/testing/messages.proto

```diff
@@ -212,3 +212,41 @@ message LoadBalancerStatsResponse {
   int32 num_failures = 2;
   map<string, RpcsByPeer> rpcs_by_method = 3;
 }
+
+// Request for retrieving a test client's real time stats.
+message LoadBalancerRealTimeStatsRequest {}
+
+// Real-time stats for RPCs sent by a test client.
+message LoadBalancerRealTimeStatsResponse {
+  // The real-time total number of RPCs issued.
+  int32 num_rpcs_started = 1;
+  // The real-time total number of RPCs completed successfully for each peer.
+  map<string, int32> num_rpcs_succeeded_by_peer = 2;
+  // The real-time total number of RPCs failed.
+  int32 num_rpcs_failed = 3;
+}
+
+// Configurations for a test client.
+message ClientConfigureRequest {
+
+  // Type of RPCs to send.
+  enum RpcType {
+    EMPTY_CALL = 0;
+    UNARY_CALL = 1;
+  }
+
+  // Metadata to be attached for the given type of RPCs.
+  message Metadata {
+    RpcType type = 1;
+    string key = 2;
+    string value = 3;
```

Comment:
Make this a repeated?

And even with this change, what if we want to attach multiple md keys to the RPC?

Reply:
You can do `[ {UNARY, 'key1', 'value1'}, {UNARY, 'key2', 'value2'} ]`. This really is equivalent to what we do for command line options today.